src/fightPage/resultModalPopUpFunctions.ts

```diff
@@ -0,0 +1,22 @@
+import { GameHandler } from "../gameHandlerClass";
+import { createFightPage } from "./renderFightPage";
+import { Player } from "../playerClass";
+
+export const showResultModal = (
+    screenToDisplay: HTMLDivElement,
+    startingPosition: string
+  ) => {
+    screenToDisplay.style.display = startingPosition;
+  };
+  
+export const playAgainModal = (
+    screenToDisplay: HTMLDivElement,
+    playerOne: Player,
+    playerTwo: Player) => {
+    const playAgainButton = screenToDisplay.querySelector(".modalPlayAgainButton");
+    playAgainButton?.addEventListener("click", () => {
+        const app = document.querySelector("#app") as HTMLDivElement;
+        const game = new GameHandler(playerOne, playerTwo);
+        setTimeout(() => createFightPage(app, game), 2000);
+    });
```

Comment:
I would also rethink if we want the game to start again with the same pokemons - maybe it would be a better idea to redirect the players to the choose pokemons page

Reply:
I changed createFightPage() to createChoosePage(). Aa a result players should be moved to ChoosePage.

Reply:
I would go even further and just reset to the main page, just like in pokemon quiz project 🙃 in order to be 100% sure that no state were left from last fight :P

Reply:
'renderApp()' will be triggered when user pushes play again button.